openhands/agenthub/codeact_agent/codeact_agent.py

```diff
@@ -340,6 +339,7 @@ def get_observation_message(
     def reset(self) -> None:
         """Resets the CodeAct Agent."""
         super().reset()
+        self.pending_actions.clear()
```

Comment:
this is interesting... it seems a valid bug fix!

Reply:
Agree, it seems fair, in `reset`